### ISSUE TITLE
Add open source license header

### ISSUE THREAD

**Issue by lukas-monet** [amosproj/amos2023ws03-gui-frame-diff#262]
## User story
1. As a user
2. I want that the open source license is added properly to the source code 
3. So that I can reuse the code

## Acceptance criteria
* every source code file has the open source license header

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
